docs/modules/ROOT/pages/additional/index.adoc

```diff
@@ -49,3 +49,24 @@ docker-compose -f <PROJECT_PATH>/ticker-backend/etc/docker-compose/docker-compos
 NOTE: Quarkus processes and optimizes beans at *build-time*, unlike traditional runtime dependency injection models. Thanks to this: Only the actually used classes are included in the final application, while unnecessary ones are automatically removed.
 
 NOTE: MP Rest Client can only be configured at *build-time*, they cannot be dynamically registered or injected at runtime, therefore at build-time it can be packaged with a `Ticker core` module.
+
+== Validating job configuration existence with test
+
+We provide an abstract test class which can be extended on project side to validate that all expected jobs have been configured.
+
+.Using of AbstractQuarkusTest class
+[source,java]
```

Comment:
a maven függőséget is beleraknám a példába + nem látom, hogy készülne test-jar a maven central-ra, amire dependálni kellene itt

Reply:
https://github.com/i-Cell-Mobilsoft-Open-Source/ticker-backend/pull/45